ServerCore/Pages/Submissions/Index.cshtml.cs

```diff
@@ -102,12 +104,20 @@ public async Task<IActionResult> OnPostAsync(int puzzleId, string submissionText
             // Create submission and add it to list
             Submission submission = new Submission
             {
-                SubmissionText = submissionText,
                 TimeSubmitted = DateTime.UtcNow,
                 Puzzle = PuzzleState.Puzzle,
                 Team = PuzzleState.Team,
                 Submitter = LoggedInUser,
             };
+            if (Puzzle.IsFreeform)
+            {
+                submission.UnformattedSubmissionText = submissionText;
+            }
+            else
+            {
+                submission.SubmissionText = submissionText;
+            }
+
             submission.Response = await _context.Responses.Where(
```

Comment:
This response means nothing in the freeform case, why look it up?

Reply:
I had the general thought that there could be freeform puzzles that still have a single final answer like Twice from PD2020. Does that sound reasonable? (Also looking at this again, I realized I wasn't formatting freeform answers for this check; fixed that)

Reply:
With the formatting pass, makes sense.